src/Bazhenov/Task5/Lighting/Light.cpp

```diff
@@ -0,0 +1,24 @@
+#include "Light.hpp"
+#include "../Geometry/Geometry.hpp"
+
+namespace Bazhenov {
+
+QVector3D ambient(const float strength, const QVector3D light_color) {
```

Comment:
```suggestion
[[nodiscard]] auto ambient(const float strength, const QVector3D light_color) noexcept {
```

и в других функциях, где не бросаются исключения и компилятор сам может вывести тип.